### PR TITLE
module: add `ak` script

### PR DESCRIPTION
**Pull request by Ma27**
This was made possible by d85dacb6c27a09f3f3df88e555148d884701ee3c which allows to directly use `manage.py`. That script is effectively used whenever the `ak` command is referenced in the docs, e.g. to set a new password for the superuser or to send a test email.

This needs to run as the same (dynamic) user and with the same env file, otherwise `manage.py` exits early. To achieve that, I decided to use `systemd-run(1)` because now the invocation can be configured the same way as services are.

cc @WilliButz @fpletz

### CODE REVIEW

**Comment by Ma27** [comment]
OK, pushed an update that solves a few issues:

* using `--collect` to remove the transient unit afterwards. Otherwise, my monitoring alerts me for failed units just because I mistyped a command
* the workdir `/var/lib/authentik` is needed for a few commands. The chdir only works though when also specifying the StateDirectory.

File: module.nix [hunk 1]
```diff
@@ -125,6 +141,20 @@ in
...
       cfg = config.services.authentik;
+
+      # Passed to each service and to the `ak` wrapper using `systemd-run(1)`
+      serviceDefaults = {
+        DynamicUser = true;
+        User = "authentik";
+        EnvironmentFile = mkIf (cfg.environmentFile != null) [ cfg.environmentFile ];
+      };
+      akOptions = flatten (mapAttrsToList
+        (k: vs: map (v: "-p ${k}=${if isBool v then boolToString v else toString v}") (toList vs))
```
Comment (WilliButz) [inline module.nix:152]:
```suggestion
      akOptions = flatten (mapAttrsToList
        # map properties from the authentik serviceConfig to command line parameters passed to systemd-run(1)
        (k: vs: map (v: "--property ${k}=${if isBool v then boolToString v else toString v}") (toList vs))
```

**Review by WilliButz — COMMENTED** [review]
Thank you for the addition :+1: 
I suggest adding a short comment about the less obvious part of `akOptions` and maybe using `--property` instead of `-p` to make it a bit easier to understand what's going on :)

File: module.nix (hunk 1 above)
Comment (Ma27) [reply module.nix]: Good point. Added another sentence to explain why we're doing this as well :)

**Review by WilliButz — APPROVED** [review]

**Pull request merged**